src/app/public/plugin-resources/code-examples/popover/programmatic/popover-demo.component.ts

```diff
@@ -19,8 +19,16 @@ export class PopoverDemoComponent {
 
   public popoverController = new Subject<SkyPopoverMessage>();
 
+  private popoverOpen: boolean = false;
+
+  public onPopoverOpen(isOpen: boolean): void {
```

Comment:
Seems a little odd that we are calling this `onPopoverOpen` but also using it on close. Could we do something like `onPopoverStateChange` or similar?

Reply:
Yep good call. Done.